util/ci

```diff
@@ -57,6 +57,8 @@ when %w(before_script)
     run('gem', %w(list --details))
     run('gem', %w(env))
   else
+    run('git', %W(reset --hard origin/#{ENV["BDV"]})) if ENV["BDV"]
```

Comment:
What `BDV` stands for?

Reply:
`BUNDLER_VERSION`, wanted to be consistent with the `RGV` variable. But being consistent in crypticness is not very important :rofl:.

We can't use `BUNDLER_VERSION` though since that variable is used by `bundler` and `rubygems` for other things.

Reply:
haha, yeah 🤔  ok lets keep it like that, i guess naming it `BV` make as much sense as `BDV` 🤷‍♂️

Reply:
Yeah, I don't really have a strong opinion. Maybe we can also use something more clear and verbose like `BUNDLER_CHECKOUT`?

Reply:
Now that i know what `BDV` stands for  i'm ok with it for consistency with `RGV. Maybe we can make both more clear and verbose in the future.

Reply:
Sounds good!